debezium-schema-generator/src/main/java/io/debezium/schemagenerator/maven/OpenApiGeneratorMojo.java

```diff
@@ -42,10 +42,10 @@
 @Mojo(name = "generate-openapi-spec", defaultPhase = LifecyclePhase.PREPARE_PACKAGE)
 public class OpenApiGeneratorMojo extends AbstractMojo {
 
-    @Parameter(property = "openapi.generator.format")
+    @Parameter(defaultValue = "openapi", property = "openapi.generator.format")
```

Comment:
What about also renaming the property name from `openapi.generator.format` to `schema.format`?
The schema must not be an OpenAPI flavor.

Reply:
And maybe also rename the class from `OpenApiGeneratorMojo` to `SchemaGeneratorMojo`?

Reply:
I think the Maven plug-in plug-in will pick up all the Mojo classes, no matter how they are named.